src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -777,5 +782,22 @@
                 </ScrollViewer>
             </PivotItem>
         </Pivot>
+        <VisualStateManager.VisualStateGroups>
+            <VisualStateGroup>
+                <VisualState>
+                    <VisualState.StateTriggers>
+                        <!--
+                            Official guidance states that 640 is the breakpoint between small and medium devices.
+                            Since MinWindowWidth is an inclusive range, we need to add 1 to it.
+                        -->
+                        <AdaptiveTrigger MinWindowWidth="641" />
+                    </VisualState.StateTriggers>
+
+                    <VisualState.Setters>
+                        <Setter Target="ControlPreview.Width" Value="640" />
+                    </VisualState.Setters>
+                </VisualState>
+            </VisualStateGroup>
+        </VisualStateManager.VisualStateGroups>

```

Comment:
Omg you are completely right, we don't need this visual state manager at all! Thanks :)

Reply:
Had to add this back in because we now change the horizontal alignment with this, and for some reason that causes the preview to disappear into oblivion unless we also set the width